lib/py/src/server/TReverseTunnelServer.py

```diff
@@ -0,0 +1,77 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from six.moves import queue
+from thrift.transport import TTransport
+#from thrift.transport.TTransport import TTransportBase, TTransportException, TServerTransportBase
+
+class TReverseTunnelServer(TTransport.TTransportBase):
+    """A transport that allows RPC Servers use Client Transports.
+
+    This useful for reverse connections, where
```

Comment:
Grammar and structure fixes to this whole section:

```
A transport that allows RPC servers to use client transports.

This is useful for "reverse connections" where a primary server
passes RPC calls to secondary servers attached to it. One example
would be an RPC service in a private network that doesn't allow
public traffic.

The current implementation of this Transport is compatible with
ThreadedPoolServer and other concurrent servers as it blocks calls
to accept() based on the number of active connections.

The idea is that for a server, a flush means a finished RPC call. This
isn't mentioned in the specification, but is a good heuristic for signalling
the accept() caller.
```

Reply:
I made some changes but could you be more specific, specially on the structure?

Reply:
I mean my version fixes some typos and grammar issues, and it reads better (to me, at least). If you don't have any objections to it, can you use it as is?

Reply:
Ahh of course, i thought it was a quote not a suggestion to use that.